projects/scion/toolkit.internal/widgets/src/tabbar/tab.directive.ts

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) 2018-2019 Swiss Federal Railways
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ *  SPDX-License-Identifier: EPL-2.0
+ */
+
+import { Directive, Input, OnDestroy, TemplateRef, ViewContainerRef, ViewRef } from '@angular/core';
+
+/**
+ * Use this directive to model a tab item for {SciTabbarComponent}.
+ * The host element of this modelling directive must be a <ng-template>.
+ *
+ * Tab content is constructed lazily when displayed for the first time. Tab content is not destroyed when selecting another tab.
+ *
+ * ---
+ * Example usage:
+ *
+ * <sci-tabbar>
+ *   <ng-template sciTab [label]="contact.name" *ngFor="let contact of contacts$ | async">
+ *     <app-contact-list-item [contact]="contact"></app-contact-list-item>
+ *   </ng-template>
+ * </sci-tabbar>
+ */
```

Comment:
Please use a neutral example, as in `SciTabDirective`.
```
<sci-tabbar>
  <ng-template sciTab [label]="item.label" *ngFor="let item of items$ | async">
   ...
  </ng-template>
</sci-tabbar>
```

Reply:
done